app/uk/gov/hmrc/cdsreimbursementclaim/services/ClaimTransformerService.scala

```diff
@@ -485,6 +482,36 @@ object DefaultClaimTransformerService {
       case None                   => Invalid(NonEmptyList.one("could not find consignee details"))
     }
 
+  def makeAccountDetails(maybeAccountDetails: Option[List[AccountDetails]]): Validation[List[AccountDetail]] =
+    maybeAccountDetails match {
+      case Some(accountDetails: Seq[AccountDetails]) =>
+        Valid(accountDetails.map { details =>
+          AccountDetail(
+            accountType = details.accountType,
+            accountNumber = details.accountNumber,
+            EORI = details.eori,
+            legalName = details.legalName,
+            contactDetails = details.contactDetails.map { cd =>
+              ContactInformation(
+                contactPerson = cd.contactName,
+                addressLine1 = cd.addressLine1,
+                addressLine2 = cd.addressLine2,
+                addressLine3 = cd.addressLine3,
+                street = cd.addressLine4,
+                city = None,
+                countryCode = cd.countryCode,
+                postalCode = cd.postalCode,
+                telephoneNumber = cd.telephone,
+                faxNumber = None,
+                emailAddress = cd.emailAddress
+              )
+            }
+          )
+        })
+      case None                                      =>
+        Invalid(NonEmptyList.one("could not find account details"))
```

Comment:
Not having any account details specified is not an error. I don't think we should be failing validation in this case.

Reply:
Agree. I've changed this to return an `Option` instead.